src/vcpkg/portlint.cpp

```diff
@@ -22,6 +24,36 @@ namespace vcpkg::Lint
         return original_scheme;
     }
 
+    Status check_usage_forgot_to_install(Filesystem& fs, const SourceControlFileAndLocation& scf, Fix fix) {
+        auto portfile_path =  scf.source_location / "portfile.cmake";
+        auto usage_path = scf.source_location / "usage";
+        if (!fs.exists(usage_path, VCPKG_LINE_INFO)) {
+            return Status::Ok;
+        }
+        // Try find the `file( ..... usage"` pattern
+        static const std::regex MATCH_USAGE_REGEX{R"###(file\([^)]*usage)###", std::regex_constants::nosubs | std::regex_constants::icase};
```

Comment:
Can we do that with std::string::find? Because std::regex is very slow and its usage was removed all over the vcpkg codebase.

Reply:
Sure